README.md

```diff
@@ -1,18 +1,83 @@
+
+From the project root, you can use the following openssl command:
+
+```
+openssl req -new -x509 -days 365 -nodes -sha256 -out lib/modules/apostrophe-salesforce-connect/connect.cer -keyout lib/modules/apostrophe-salesforce-connect/connect.key -days 3650
+```
+
+### Project configuration
+
+The bundle module requires several project-level configurations and also allows for additional optional project-level configurations to enable additional features. A private repository can be a reasonable place for these settings, but if you do not wish some of these settings to be present in version control or you want them to vary from server to server, [you can use the `data/local.js` file](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/deployment#minifying-assets). You can also use environment variables, via `process.env.YOUR_ENV_VAR_NAME`. The queries and other non-sensitive information could instead be configured in the `app.js` module declaration.
+
+The `adminUsername`, `adminPasswordAndToken`, and `adminLoginUrl` are all required. These should relate to an admin-level user in your Salesforce instance and are used to make the initial SOQL query and subsequent user-specific queries via OAuth.
+
+`adminPasswordAndToken` is made up of both the user's password and their security token, in that order, as a single string. [Salesforce has documentation on getting this token.](https://help.salesforce.com/articleView?id=user_security_token.htm&type=5) For example, with a password of `password` and token of `0a1b2c3d4e5f`, the final value would be `password0a1b2c3d4e5f`.
+
+If the `redirectionUrl` is configured, after end users sign in on the Salesforce page, they will be redirected to the `redirectionUrl` value rather than back to the ApostropheCMS website.
+
+`experienceQueries` is an array of objects with both general and user-specific queries. You may have multiple such configurations in order to build out varying user experiences. For example, one query set might capture "experiences" for Salesforce "Groups" the person is in, then another might create additional "experience" values for their related "Products." These can exist together to allow more options and content possibilities.
+
+All properties of the `experienceQueries` object are required. Within each `experienceQueries` object, the options are:
+
+- `soql`: The general query to capture names and unique IDs for "experiences."
+- `labelField`: The field in the `soql` query that returns a human-readable label for the experience.
+- `idField`: The field in the `soql` query that returns a unique ID value for the experience.
+- `userSoql`: An SOQL query that requests information about a specific user, by their account ID, with returned values that match the `labelField` and `idField` values. This should include the string `:AccountId`, which will be replaced with the actual user's account ID.
```

Comment:
This doesn't really make clear the relationship between `soql` and `userSoql`.

How about:

Similar to `soql`, except that it must return only the labels and unique ids for those experiences that currently apply to a specified user.